packages/react-native-worklets/src/initializers/initializers.native.ts

```diff
@@ -72,12 +72,12 @@ export function setupConsole(boundCapturableConsole: typeof console) {
   'worklet';
   // @ts-ignore TypeScript doesn't like that there are missing methods in console object, but we don't provide all the methods for the UI runtime console version
   globalThis.console = {
-    assert: runOnJS(boundCapturableConsole.assert),
-    debug: runOnJS(boundCapturableConsole.debug),
-    log: runOnJS(boundCapturableConsole.log),
-    warn: runOnJS(boundCapturableConsole.warn),
-    error: runOnJS(boundCapturableConsole.error),
-    info: runOnJS(boundCapturableConsole.info),
+    assert: (...args) => scheduleOnRN(boundCapturableConsole.assert, ...args),
+    debug: (...args) => scheduleOnRN(boundCapturableConsole.debug, ...args),
+    log: (...args) => scheduleOnRN(boundCapturableConsole.log, ...args),
+    warn: (...args) => scheduleOnRN(boundCapturableConsole.warn, ...args),
+    error: (...args) => scheduleOnRN(boundCapturableConsole.error, ...args),
+    info: (...args) => scheduleOnRN(boundCapturableConsole.info, ...args),
```

Comment:
```suggestion
    assert: scheduleOnRN.bind(null, boundCapturableConsole.assert),
    debug: scheduleOnRN.bind(null, boundCapturableConsole.debug),
    log: scheduleOnRN.bind(null, boundCapturableConsole.log),
    warn: scheduleOnRN.bind(null, boundCapturableConsole.warn),
    error: scheduleOnRN.bind(null, boundCapturableConsole.error),
    info: scheduleOnRN.bind(null, boundCapturableConsole.info),
```

maybe use `.bind` here as well if it works in worklets?